drivers/mtd_spi_nor/mtd_spi_nor.c

```diff
@@ -374,10 +398,14 @@ static inline void wait_for_write_complete(const mtd_spi_nor_t *dev, uint32_t us
 #endif
     } while (1);
     DEBUG("wait loop %u times, yield %u times", i, j);
-    if (IS_ACTIVE(ENABLE_DEBUG) && IS_ACTIVE(MODULE_XTIMER)) {
-        diff = xtimer_now_usec() - diff;
-        DEBUG(", total wait %"PRIu32"us", diff);
-    }
+#if IS_ACTIVE(ENABLE_DEBUG) && (IS_USED(MODULE_ZTIMER_USEC) || IS_USED(MODULE_XTIMER))
```

Comment:
```suggestion
#if IS_ACTIVE(ENABLE_DEBUG)
```
make this easier to read

Reply:
addressed